<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,66,0);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;.//resource//default.png&quot;);
background : transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;.//resource//siren.png&quot;);
background : transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>310</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;.//resource//babycry.png&quot;);
background : transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>310</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;.//resource//scream.png&quot;);
background : transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;.//resource//biground.png&quot;);
background : transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>571</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;.//resource//logo.png&quot;);
background : transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent</string>
   </property>
   <property name="text">
    <string>기본 감지</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent</string>
   </property>
   <property name="text">
    <string>사이렌 감지</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent</string>
   </property>
   <property name="text">
    <string>울음소리 감지</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>560</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent</string>
   </property>
   <property name="text">
    <string>비명소리 감지</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;.//resource//backbutton.png&quot;);
background : transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>toMain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1227</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>1306</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>todefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>-89</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>tosiren()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>830</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>tobabycry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>847</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>toscream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1070</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>1075</x>
     <y>846</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toMain()</slot>
  <slot>todefault()</slot>
  <slot>tosiren()</slot>
  <slot>tobabycry()</slot>
  <slot>toscream()</slot>
  <slot>tonowtrain()</slot>
 </slots>
</ui>
